ui/page/root/wallet_settings_page.go

```diff
@@ -764,10 +708,6 @@ func (pg *WalletSettingsPage) HandleUserInteractions() {
 		pg.ParentNavigator().Display(s.NewStatPage(pg.Load))
 	}
 
-	if pg.proposalNotif.Changed() {
-		pg.WL.SelectedWallet.Wallet.SaveUserConfigValue(sharedW.ProposalNotificationConfigKey, pg.proposalNotif.IsChecked())
-	}
-
```

Comment:
this should now be a general app setting

Reply:
Yeah i know but i think it will be best to add it from another PR. 
Reason is it will require modifying setting page UI codes and libwallet.

Reply:
Okay please create a follow up issue